boards/cats_rev1Pro/Core/Src/config/globals.c

```diff
@@ -46,6 +46,15 @@ const ICM20601 ICM3 = {
     .gyro_dps = ICM20601_GYRO_RANGE_2000_DPS,
 };
 
+SPI_BUS SPI_ACCEL = {.cs_port = CS_ACC_GPIO_Port, .cs_pin = CS_ACC_Pin, .spi_handle = &hspi1, .cs_type = LOW_ACTIVE};
+
+const H3LIS100DL ACCEL = {
```

Comment:
why is this one called ACCEL and the other ones are named after their model? I would probably rename the others to IMU0,1... and this one can be ACC_HIGH_G or something like that.

Otherwise name this one by its model also

Reply:
The other are not named after the model, the sensor is ICM and they are defined as IMU (Inertial measurement unit)

Reply:
Wait actually they are now named after the model, I think this is new but idk. Either way we need to change things up with the sensors anyway